MaintenanceManager/CHANGELOG.md

```diff
@@ -16,6 +16,14 @@ All notable changes to this RDK Service will be documented in this file.
 
 * For more details, refer to [versioning](https://github.com/rdkcentral/rdkservices#versioning) section under Main README.
 
+## [1.0.22] - 2023-09-08
+### Changed
+- Renamed partnerProvisioningContext to deviceInitializationContext for new WhoAmI specification
+
+## [1.0.21] - 2023-09-08
+### Security
+- Brought SecManager code under ENABLE_WHOAMI compile flag
+
```

Comment:
Do we need to bump the API version twice from `20` -> `22`? Both changes will take effect atomically so there will be no clients that use the `21` version of this API. Is it because they are different categories?